balancer/weightedroundrobin/weightedroundrobin.go

```diff
@@ -19,11 +19,41 @@
 // Package weightedroundrobin defines a weighted roundrobin balancer.
 package weightedroundrobin
 
-// Name is the name of weighted_round_robin balancer.
-const Name = "weighted_round_robin"
+import "google.golang.org/grpc/attributes"
+
+const (
+	// Name is the name of weighted_round_robin balancer.
+	Name = "weighted_round_robin"
+
+	// Attribute key used to store AddrInfo in the Attributes field of
+	// resolver.Address.
+	attributeKey = "/balancer/weightedroundrobin/addrInfo"
```

Comment:
Make a type, so only this package can get/set the value?

Reply:
Done. Thanks.